### PR TITLE
feat: implement handle screen change

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

**Review by NerdbordMentor — APPROVED** [review]
## Code Review Report

### Task Goal
The goal of the task is to be able to submit a sandwich order. 

### Requirements
1. Clicking on the **Place order** button should console.log form payload when form validation passes.
2. Clicking on the **Place order** button should redirect to the **Success screen**.

### Code Changes Review

The code changes seem to cover the task requirements.

1. The **Place order** button now logs the form payload to the console when form validation passes. This is done in the `onSubmit` function:

```diff
+ console.log('Success, Form Data:', data)
```

2. The **Place order** button now triggers a redirect to the **Success screen** after a delay of 1 second. This is done using the `setTimeout` function:

```diff
+ setTimeout(onPlaceOrder, 1000)
```

The `onPlaceOrder` function is passed as a prop to the `PaniniCreator` component:

```diff
+ const PaniniCreator = ({ onPlaceOrder }: PaniniCreatorProps) => {
```

This function is expected to handle the redirection to the **Success screen**.

In conclusion, the code changes seem to meet the task requirements. However, it's important to note that the actual implementation of the `onPlaceOrder` function and its effect (i.e., redirection to the **Success screen**) are not visible in the provided code changes. Therefore, a complete review would require a look at how this function is implemented elsewhere in the codebase.
